.github/workflows/release.yml

```diff
@@ -0,0 +1,112 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - ci/*
+      - release/*
+    tags:
+      - v*
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  # This enables the workflow to create GitHub releases
+  contents: write
```

Comment:
can we grant write access in the release job only?

```yaml
jobs:
  release:
    permissions:
      contents: write
```

Reply:
There should be a way to do that. I'll check and update accordingly today.

Reply:
Of course! I'm on it.

Reply:
Done - https://github.com/filecoin-project/lotus/pull/11785/commits/b3c24acf4bdfbf2e9331916f06adc7be3d5b76ea ✅